app/src/main/java/fr/free/nrw/commons/customselector/ui/adapter/ImageAdapter.kt

```diff
@@ -204,8 +199,6 @@ class ImageAdapter(
                             uploadingContributionList
                         )
                         _isLoadingImages.value = false
-                        // If the position is already visited, that means the image is already present
```

Comment:
This PR removes a lot of comments here and below, is that intended?

By the way it seems to be working, I will test more tomorrow.